cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -84,6 +91,29 @@ def _create_perfect_calibration(device: cirq.Device) -> calibration.Calibration:
     return calibration.Calibration(calibration=snapshot, metrics=all_metrics)
 
 
+def load_median_device_calibration(processor_id: str) -> calibration.Calibration:
+    """Loads a median Calibration for the given device.
+
+    Real calibration data from Google's 'rainbow' and 'weber' devices has been
+    saved in Cirq. The calibrations selected are roughly representative of the
+    median performance for that chip.
+
+    Args:
+         processor_id: name of the processor to simulate.
+
+    Raises:
+        ValueError: if processor_id is not a supported QCS processor.
```

Comment:
nit: this indentation looks like it is off by a space?

Reply:
This was an odd one - all of the `Args:` sections in this file used a 5-space indent. Fixed to make it 4-space everywhere.